helm/polaris/NOTICE

```diff
@@ -0,0 +1,2 @@
+Apache Polaris (incubating)
+Copyright 2024 The Apache Software Foundation
```

Comment:
Do we even have to have this NOTICE in Helm source distributions? IMHO, it does not add anything that is not already covered by LICENSE.

Reply:
I guess we have to have the default NOTICE per https://www.apache.org/legal/src-headers.html#notice

... but the standard text appears to be a bit more verbose:

```
Apache [PRODUCT_NAME]
Copyright [XXXX-XXXX] The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (/).
```